base/Commands/DemoCommand.php

```diff
@@ -48,49 +50,24 @@ class DemoCommand extends QtCommand
      * @return void
      * @throws \Quantum\Exceptions\DiException
      */
+    
     public function exec()
-    {
-        $postsCollection = ([
-            [
-                'id' => 1,
-                'title' => 'Walt Disney',
-                'content' => 'The way to get started is to quit talking and begin doing.',
-                'author' => 'admin@qt.com',
-                'image' => 'walt-disney-9252507491.jpg',
-                'updated_at' => '07/22/2021 16:31',
-            ],
-            [
-                'id' => 2,
-                'title' => 'James Cameron',
-                'content' => 'If you set your goals ridiculously high and it is a failure, you will fail above everyone else success.',
-                'author' => 'admin@qt.com',
-                'image' => null,
-                'updated_at' => '05/08/2021 23:13',
-            ],
-        ]);
-
-        $this->renderContent($postsCollection, 'posts');
-
-        $usersCollection = ([
-            [
-                'id' => '1',
-                'firstname' => 'Admin',
-                'lastname' => 'Hunter',
-                'role' => 'admin',
-                'email' => 'admin@qt.com',
-                'password' => '$2y$12$4Y4/1a4308KEiGX/xo6vgO41szJuDHC7KhpG5nknx/xxnLZmvMyGi',
-                'activation_token' => '',
-                'remember_token' => '',
-                'reset_token' => '',
-                'access_token' => '',
-                'refresh_token' => '',
-                'otp' => '',
-                'otp_expires' => '',
-                'otp_token' => '',
-            ],
-        ]);
-
-        $this->renderContent($usersCollection, 'users');
+    { 
+        $faker = Factory::create();
+        $roles = ['admin', ''];
+        $usersCollectionData = [];
+
+        for ($i = 0; $i < 2; $i++) {
```

Comment:
Remove the loop and call the method 2 times by passing role=admin for admin and null for guest